rasa/cli/run.py

```diff
@@ -60,6 +60,20 @@ def run(args: argparse.Namespace):
     import rasa.run
 
     args.model = get_validated_path(args.model, "model", DEFAULT_MODELS_PATH)
```

Comment:
I think we should move this line in the `if not args.enable_api` block and set it to `None` or so.

Reply:
Cause atm if fails there is no directory `models` and you try to run it with `--enable-api`

Reply:
We can move it insight, but then the following might happen:
You run `rasa run --enable-api -m non-existing-model`. We are not validating if `non-existing-model` is a valid path. Thus the output will look like:
``` 
No chat connector configured, falling back to the REST input channel. To connect your bot to another channel, read the docs here: https://rasa.com/docs/rasa/user-guide/messaging-and-voice-channels
2019-06-05 11:27:49 INFO     root  - Starting Rasa Core server on http://localhost:5005
2019-06-05 11:27:49 WARNING  rasa.core.agent  - No valid configuration given to load agent.
2019-06-05 11:27:49 WARNING  root  - Agent could not be loaded with the provided configuration. Load default agent without any model.
```
Not sure if that is clear to the user.

Reply:
Good point. Should we try to create the directory if it does not exist?

Reply:
Updated the PR.